esrally/track/loader.py

```diff
@@ -189,15 +191,41 @@ def load_track(cfg):
     :return: The loaded track.
     """
     repo = track_repo(cfg)
-    return _load_single_track(cfg, repo, repo.track_name)
+    return _load_single_track(cfg, repo, repo.track_name, install_dependencies)
 
 
-def _load_single_track(cfg, track_repository, track_name):
+def _install_dependencies(dependencies):
+    def _cleanup():
+        # fully destructive is fine, we only allow one Rally to run at a time and we will rely on the pip cache for download caching
+        shutil.rmtree(paths.libs(), onerror=_trap)
```

Comment:
I've used it and `console.info` would be nice here, in the same way that "Installing track dependency X" is nice